src/App.tsx

```diff
@@ -1,26 +1,55 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { todosSlice } from './features/todos';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(result => {
+        dispatch(todosSlice.actions.storeTodos(result));
```

Comment:
The `storeTodos` reducer in `todosSlice` should not directly return a new array. Instead, it should update the state immutably. Consider using the spread operator to update the state: `state => [...state, ...payload]`.